test/test_dataloader.py

```diff
@@ -417,6 +417,7 @@ def test_growing_dataset(self):
         self.assertEqual(len(dataloader_shuffle), 5)
 
     @unittest.skipIf(not TEST_CUDA, "CUDA unavailable")
+    @skipIfRocm
```

Comment:
So these tests don't pass anymore after switching to the new target?

Reply:
Correct. We are looking into this to resolve.